src/parser.js

```diff
@@ -34,38 +46,113 @@ export function containIncludeCommand(commands = []) {
         return reg.test(command.trim());
     })
 }
+/** Parse the command label and return key-value object
+ * @example
+ *      [import,title:"<thetitle>",label:"<thelabel>"](path/to/file.ext)
+ * @param {string} label
+ * @return {Array}
+ */
+export function parseVariablesFromLabel(label) {
+    var keyvals = {
+        "title":undefined,
+        "id":undefined,
+        "marker":undefined
+    };
+    for(var key in keyvals) {
```

Comment:
Please use `Object.keys(keyvals).forEach(key => { ... })` instead of for..in.
[for...in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in "for...in") has a trap.

Related:

[Rule guard-for-in - ESLint - Pluggable JavaScript linter](http://eslint.org/docs/rules/guard-for-in "Rule guard-for-in - ESLint - Pluggable JavaScript linter")